### DISCUSSION TITLE
Problem with a matching result

### DISCUSSION THREAD

**Discussion by DesislavaStrahilova** [cssbattle/forum#275] [Bugs reports]
Hi! 

I have a problem about css battle 19, target 114 and hope you will help me!
Two days ago I had made it with 99.9% match. 
Today I decided to try for 100% match, but for some reasons now it shows  79.6% match. 
I had copied and pasted my old code /from 21 june, with 99.9% match/, but it's useless - every time the result remains 79.6%. 

What is the reason for that and how can I correct it? 

Thank you in advance!

**Comment by chinchang** [comment]
Hi @DesislavaStrahilova You probably are seeing the old target. You can do a hard refresh to see the updated one. Read more here - https://github.com/cssbattle/forum/discussions/273#discussion-4165561

**Reply by DesislavaStrahilova** [reply]
Thank you very much for the quick answear!

I tried, but didn't succeed ... Again ... I suppose this will be my end result for this target. Whatever ...

Thank you once again!

**Reply by Beowolve** [reply]
This should work:

* Go to https://cssbattle.dev/targets/114.png
* Press F5 (reload)

or

delete your browser cache

**Reply by DesislavaStrahilova** [reply]
With your advice - this time 100% match. 
Thank you!
Have a great day!
